src/main/java/com/project/controller/UserController.java

```diff
@@ -25,29 +24,29 @@ public UserController(UserService userService) {
     }
 
     @GetMapping
-    public ResponseEntity<List<User>> getUsers(){
+    public ResponseEntity<List<User>> getUsers() {
```

Comment:
If you have `fetchUser` then this should be called `fetchUsers`.

Reply:
renamed getUsers to fetchAllUsers.
made corresponding changes to ApplicationController class